backend/src/main/java/com/it/p/lodz/pl/masi/entities/PositionEntity.java

```diff
@@ -115,4 +115,7 @@ public void setResolvedTestsById(Collection<ResolvedTestEntity> resolvedTestsByI
     public void setTestsById(Collection<TestEntity> testsById) {
         this.testsById = testsById;
     }
+    public void addTestToPosition(TestEntity test){
```

Comment:
Jeśli zmienisz `TestService` tak jak napisałem, to ta metoda nie będzie potrzebna :)